test/kubernetes_test.go

```diff
@@ -3,72 +3,159 @@
 package test
 
 import (
-	"io/ioutil"
-	"log"
 	"testing"
 	"time"
 
 	"github.com/mholt/caddy"
+	"github.com/miekg/coredns/middleware/test"
```

Comment:
Please sort the coredns import before the external ones, so: stdlib, coredns, rest